tests/e2e/specs/shopper/shopper-wc-blocks-checkout-purchase.spec.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopperWCP,
+	merchantWCP,
+	describeif,
+	RUN_WC_BLOCKS_TESTS,
+} from '../../utils';
+
+const shippingDetails = config.get( 'addresses.customer.shipping' );
+const productName = config.get( 'products.simple.name' );
+
+import {
+	fillCardDetailsWCB,
+	confirmCardAuthentication,
+} from '../../utils/payments';
+
+describeif( RUN_WC_BLOCKS_TESTS )(
+	'WooCommerce Blocks > Successful purchase',
+	() => {
+		beforeAll( async () => {
+			await merchant.login();
+			await merchantWCP.addNewPageCheckoutWCB();
+			await merchant.logout();
+		} );
+
+		afterAll( async () => {
+			await shopper.logout();
+		} );
+
+		it( 'using a basic card', async () => {
+			await shopper.login();
+			await shopper.goToShop();
+			await shopper.addToCartFromShopPage( productName );
+			await shopperWCP.openCheckoutWCB();
+			await shopperWCP.fillShippingDetailsWCB( shippingDetails );
+
+			// Fill CC details and purchase the product
+			const card = config.get( 'cards.basic' );
+			await fillCardDetailsWCB( page, card );
+			await expect( page ).toClick( 'button', { text: 'Place Order' } );
+			await page.waitForSelector( 'div.woocommerce-order' );
+			await expect( page ).toMatch( 'p', {
+				text: 'Thank you. Your order has been received.',
+			} );
+		} );
+
+		it( 'using a 3DS card', async () => {
+			await shopper.goToShop();
```

Comment:
Just learning for myself: how it works without the `shopper.login();`? 
It stays logged in after `using a basic card`, correct?

Reply:
Works the same, yes it might be logged in in the next scenario. I'll move `shopper.login();` to beforeAll functions.